### ISSUE TITLE
SharpApp Telemetry Hosts File being blocked/quarantined by Windows Defender

### ISSUE THREAD

**Issue by ghnameless** [builtbybel/sharpapp#39]
Windows 10 is blocking edits of my hosts file thru SharpApp. This has never happened before, it only started today. 

SharpApp version 0.46.0

![sharpapp_quarantined](https://user-images.githubusercontent.com/68933609/88748220-0ef3c880-d183-11ea-8a02-5c07d8b300a9.png)

**Comment by Belim** [comment]
Windows Defender has some false positive issues lately. Does it still flag the action as FileHijack?

**Comment by ghnameless** [comment]
Yes, windows defender blocks it whenever I edit the hosts file using SharpApp (I use the spy community host). I just allowed it in my windows defender so it won't get reset to default. Not really that bothersome as I can just allow it but I might as well post the issue. Definitely a false positive though.

**Comment by Belim** [comment]
HOSTS file blocking telemetry is now flagged as a risk
https://www.bleepingcomputer.com/news/microsoft/windows-10-hosts-file-blocking-telemetry-is-now-flagged-as-a-risk/
